modules/dnn/src/layers/padding_layer.cpp

```diff
@@ -261,7 +279,8 @@ class PaddingLayerImpl CV_FINAL : public PaddingLayer
     std::vector<std::pair<int, int> > paddings;  // Pairs pad before, pad after.
     std::vector<Range> dstRanges;
     int inputDims;
-    float paddingValue;
+    double paddingValue;
```

Comment:
in ONNX parser paddingValue has the same depth as the input layer, so it can have int type. So we need to store paddingValue using some variable with dynamic type. To make things simpler I just use double type, it should be enough.